docs/tutorial-basics/extracting_prediction_tasks.md

```diff
@@ -2,13 +2,17 @@
 sidebar_position: 4
 ---
 
+import JupyterViewer from "react-jupyter-notebook";
+import nb from "../../tutorial_notebooks/tutorial-basics/extracting_prediction_tasks.ipynb";
+
 # Extracting a Prediction Task Cohort from MEDS Data
 
-To easily extract prediction tasks from MEDS Data, you can use the
-[ACES](https://eventstreamaces.readthedocs.io/en/stable/) package. This package allows you to define simple
+To easily extract prediction tasks from MEDS Data, we will use the
+[ACES](https://eventstreamaces.readthedocs.io/en/latest/) package. This package allows you to define simple
```

Comment:
Maybe just have one use of ACES be a documentation link? Or at least have them both go to the same version of the documentation.